Sources/Persistence/ServerSyncJobStore.swift

```diff
@@ -10,7 +11,13 @@ struct ServerSyncJobStore {
     }
 
     private func loadOperations() -> [ServerSyncJob] {
-        guard let operations = NSData(contentsOfFile: syncJobStoreFileName) else {
+        guard let fileURL = fileManager.urls(for: .documentDirectory, in: .userDomainMask).last else {
```

Comment:
I wonder if we should use `cachesDirectory`. But I don't know what conditions have to take place for caches to be cleared. I assume `temporaryDirectory` will be cleaned after a device restart.

One advantage of cache directory is that they won't get sync'ed with iCloud, which makes it less problematic when restoring an app. I don't know if this is something to consider at this point

Reply:
> But I don't know what conditions have to take place for caches to be cleared.
I did some research and I couldn't find the answer in the documentation stating when exactly this would happen.

We can however exclude the file from the backup by:
`filePath.setTemporaryResourceValue(true, forKey: .isExcludedFromBackupKey)`

[How do I prevent files from being backed up to iCloud and iTunes?](https://developer.apple.com/library/archive/qa/qa1719/_index.html)

What do you think?

Reply:
Resolved in e3570b0